utopia-remix/app/routes/internal.editor.projects.$id.sharing.tsx

```diff
@@ -1,14 +1,16 @@
 import type { LoaderFunctionArgs } from '@remix-run/node'
-import { json, useFetcher, useLoaderData } from '@remix-run/react'
-import type { UserDetails } from 'prisma-client'
+import { json, useFetcher, useLoaderData, useRevalidator } from '@remix-run/react'
 import React from 'react'
 import { SharingDialogContent } from '../components/sharingDialog'
 import { getProjectSharingDetails } from '../models/project.server'
-import type { SharingProjectAccessRequests } from '../stores/projectsStore'
-import { ProjectsContext, createProjectsStore } from '../stores/projectsStore'
-import { AccessLevel, asAccessLevel, type ProjectSharingDetails } from '../types'
+import { ProjectsContext, createProjectsStore, useProjectsStore } from '../stores/projectsStore'
+import type { ProjectAccessRequestWithUserDetails, ProjectSharingDetails } from '../types'
+import { AccessLevel, asAccessLevel } from '../types'
 import { ensure, requireUser } from '../util/api.server'
 import { Status } from '../util/statusCodes'
+import type { UserDetails } from 'prisma-client'
+
+const RevalidateInterval = 10_000 //ms
```

Comment:
Is this supposed to be 15 seconds from the description?

Reply:
🤦 fixed